tests/MiniProfiler.Tests/MiniProfiler.Tests.csproj

```diff
@@ -17,7 +17,13 @@
   </ItemGroup>
   <ItemGroup Condition="'$(TargetFramework)' == 'net461'">
     <PackageReference Include="sqlite" Version="3.13.0" />
+    <PackageReference Include="Npgsql" Version="4.0.5" />
     <ProjectReference Include="..\..\src\MiniProfiler.Providers.SqlServerCe\MiniProfiler.Providers.SqlServerCe.csproj" />
+    <ProjectReference Include="..\..\src\MiniProfiler.Providers.PostgreSql\MiniProfiler.Providers.PostgreSql.csproj" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp2.0'">
+    <PackageReference Include="Npgsql" Version="4.0.5" />
```

Comment:
Since it's in both - we can just move this reference to the unconditional group above.

Reply:
netcoreapp1.1 wasn't supported by the current version of Npgsql. I can add in an old version of Npgsql if it needs to work with 1.1.